<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferenceEditor</class>
 <widget class="QDialog" name="ReferenceEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор справочников</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string>Редактор справочников</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="dictionaryTView"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">background:transparent;</string>
             </property>
             <property name="text">
              <string>Выберите справочник:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="ReferenceCb">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 3px solid rgb(162, 162, 162); 
border-radius: 3px;
padding: 1px 1px 1px 1px;</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Предметы</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Активности</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Аудитории</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Пользователи</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Группы</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Сохранить</string>
             </property>
             <property name="icon">
              <iconset resource="ImageResourses.qrc">
               <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="CloseBt">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>40</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>Закрыть</string>
             </property>
             <property name="icon">
              <iconset resource="ImageResourses.qrc">
               <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QCheckBox" name="xmlConfigoxCb">
           <property name="styleSheet">
            <string notr="true">background:transparent;</string>
           </property>
           <property name="text">
            <string>xml конфигуратор</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImageResourses.qrc"/>
 </resources>
 <connections/>
</ui>
